code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -86,4 +86,36 @@
 	icon_state = "[initial(icon_state)][silenced ? "-silencer" : ""][chambered ? "" : "-e"]"
 	return
 
-
+/obj/item/weapon/gun/projectile/handgun //mime fingergun 
+	name = "hand-gun"
+	desc = "This is a stickup!"
+	icon_state = "handgun"
+	inhand_states = null
+	ammo_type = "/obj/item/ammo_casing/invisible"
+	mag_type = "/obj/item/ammo_storage/magazine/invisible"
+	cant_drop = 1
```

Comment:
We're going to start using TRUE and FALSE for bools, so it's an idea to get into the habit.